lib/_emerge/depgraph.py

```diff
@@ -2935,6 +2936,13 @@ def _changed_deps(self, pkg):
             else:
                 depvars = Package._runtime_keys
 
+            libc_deps = find_libc_deps(
```

Comment:
It's wasteful to repeat this over an over, so we really should cache the result someplace like self._frozen_config.

Reply:
This is the last TODO, will do that now.